src/hacbs/components/Commits/CommitsListRow.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { GithubIcon } from '@patternfly/react-icons/dist/js/icons/github-icon';
+import { useApplicationActions } from '../../../components/ApplicationListView/application-actions';
+import ActionMenu from '../../../shared/components/action-menu/ActionMenu';
+import ExternalLink from '../../../shared/components/links/ExternalLink';
+import { RowFunctionArgs, TableData } from '../../../shared/components/table';
+import { Timestamp } from '../../../shared/components/timestamp/Timestamp';
+import { commitsTableColumnClasses } from './CommitsListHeader';
+
+const CommitsListRow: React.FC<RowFunctionArgs<any>> = ({ obj }) => {
+  const actions = useApplicationActions(obj);
+  return (
+    <>
+      <TableData className={commitsTableColumnClasses.name}>
+        <ExternalLink href={obj.shaURL}>
+          {`${obj.sha} `}
+          <GithubIcon />
+        </ExternalLink>
+      </TableData>
+      <TableData className={commitsTableColumnClasses.branch}>
+        {obj.repoURL && obj.branch ? (
+          <ExternalLink href={`${obj.repoURL}/tree/${obj.branch}`} text={obj.branch} />
+        ) : (
+          obj.branch ?? '-'
+        )}
+      </TableData>
+      <TableData className={commitsTableColumnClasses.component}>
+        {obj.component ? (
+          <Link to={`/app-studio/component?name=${obj.component}`} title={obj.component}>
+            {obj.component}
+          </Link>
+        ) : (
+          '-'
+        )}
+      </TableData>
+      <TableData className={commitsTableColumnClasses.byUser}>{obj.user ?? '-'}</TableData>
+      <TableData className={commitsTableColumnClasses.committedAt}>
+        <Timestamp timestamp={obj.creationTime} />
+      </TableData>
+      <TableData className={commitsTableColumnClasses.status}>-</TableData>
```

Comment:
Status Column should be populated here. There could be multiple pipelineruns associated with a commit, should we show the latest pipelinerun status here?